x-pack/plugins/security_solution/public/common/lib/apm/user_actions.ts

```diff
@@ -23,6 +23,7 @@ export const BULK_RULE_ACTIONS = {
   DISABLE: `${APP_UI_ID} bulkRuleActions disable`,
   DUPLICATE: `${APP_UI_ID} bulkRuleActions duplicate`,
   EXPORT: `${APP_UI_ID} bulkRuleActions export`,
+  SCHEDULE_BACKFILL: `${APP_UI_ID} bulkRuleActions schedule backfill`,
```

Comment:
nit: this is the only remaining place where we use the term "backfil"

Reply:
done